webapp/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -9,9 +9,12 @@ import {
   User,
   UserControllerGetUsersRequest,
 } from '../../shared/generated';
-import { DispatchPage } from '../../shared/components/index';
+import { RowsTemplate } from '../../shared/components/index';
 import { useCallback } from 'react';
 import { usersApi } from '../../shared/services/ApiService';
+import { SearchContextProvider } from '../../shared/context/SearchContext';
+
+const ENTRIES_LIMIT = 15;
```

Comment:
why 15?

Reply:
If I'm not wrong we already have this limit in the back-end set to 20, in the [user.pagination.dto.ts](https://github.com/42AGV/ft_transcendence/blob/main/transcendence-app/src/user/dto/user.pagination.dto.ts) file
MAX_USER_ENTRIES_PER_PAGE = 20;
Though, I am moving this constant to /shared/constants.tsx file with the chat limit that I am working on

Reply:
It is down the max_limit. This constant is necessary so we can set de offset of our pagination queries (we fetch data ENTRIES_LIMIT by ENTRIES_LIMIT)